backend/manage.py

```diff
@@ -20,3 +20,8 @@ def main():
 
 if __name__ == '__main__':
     main()
+
+# attempt to work around the psql fatal error
```

Comment:
What was the error again, and is the error resolved now because of this?

Reply:
@anshumaantgi

Reply:
Yeah, the Error was an on-machine psql error not really a server error.  With the correct implementation of psql on the machine, the code should work fine.